packages/block-editor/src/components/link-control/style.scss

```diff
@@ -168,35 +168,38 @@ $preview-image-height: 140px;
 		align-items: flex-start;
 		margin-right: $grid-unit-10;
 		overflow: hidden;
-
-		// Force text to wrap to improve UX when encountering long lines
-		// of text, particular those with no spaces.
-		// See: https://github.com/WordPress/gutenberg/issues/33586#issuecomment-888921188
-		white-space: pre-wrap;
-		word-wrap: break-word;
+		white-space: nowrap;
+		line-height: 1.5;
 	}
 
 	&.is-preview .block-editor-link-control__search-item-header {
 		display: flex;
-		flex: 1; // fill available space.
+		flex: 1; // Fill available space.
 	}
 
 	.block-editor-link-control__search-item-details {
-		overflow: hidden; // clip to force text ellipsis.
+		overflow: hidden; // Clip to force text ellipsis.
```

Comment:
Just to be clear we changed this behaviour to allow much more text to display. This is because it better mirrors the classic editor link behaviour and also it makes it more optimised for selecting items when you need more context.

With only a small preview of text you sometimes cannot tell the different between two items. I don't think we should roll back on this behaviour. It was requested by several 3rd party contributors.